docs/remote-bazel-introduction.md

```diff
@@ -0,0 +1,228 @@
+---
+id: remote-bazel-introduction
+title: Introduction to Remote Bazel
+sidebar_label: Remote bazel introduction
+---
+
+Remote Bazel is an easily configurable way to run commands on a remote runner. You
+can think of it as dynamically spinning up a VM to execute a single command.
```

Comment:
is it only single? (don't we have steps in the api now?)

Reply:
Changed the wording a bit